Data Cleaning/Viz- Subplot Method Arguments (#38655)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-cleaning-and-visualizations.english.md

```yml
question:

>   text: 'When using MatPlotLib's global API, the following code does what in what order?:

Reviewer: Matplotlib is just capitalized, the apostrophe here in `Matplolib's` needs to be escaped, and the original wording of the question felt awkward. Here's a suggested change

```suggestion
  text: 'When using Matplotlib''s global API, what does the order of numbers mean here?:
```

  <pre>plt.subplot(1, 2, 1)</pre>
  answers:

>     - 'My figure will have one column, two rows, and I am goin to start drawing in the first (left) plot.'

Reviewer: Typo

```suggestion
    - 'My figure will have one column, two rows, and I am going to start drawing in the first (left) plot.'
```
